### DISCUSSION TITLE
how to build hoarder web image?

### DISCUSSION THREAD

**Discussion by microyahoo** [karakeep-app/karakeep#686] [Q&A] [answered]
I forked the Hoarder code locally and executed `docker build -t horder:latest -f docker/Dockerfile . -D --progress=plain` to build the Hoarder image. However, I found that my built image size is significantly different from the official image size, and it throws errors when running. Please advise how to build the web image correctly and how to resolve these errors? Thanks.
![WX20241123-010115@2x](https://github.com/user-attachments/assets/316e1004-e05d-41fa-98e7-d3fd488393a5)
![WX20241123-011055@2x](https://github.com/user-attachments/assets/90d9a8d0-a30b-48fa-8655-22f13aea96bc)

**Comment by MohamedBassem** [comment, marked as answer]
The `Dockerfile` contains multiple images (egacy web & workers, CLI, and the current all in one image. You need to tell docker which want it should build. The prod image is the `aio` image, so your command should have `--target=aio`

**Reply by microyahoo** [reply]
It comes to normal, thanks!
